src/libs/OptionsListUtils.js

```diff
@@ -82,9 +82,16 @@ function createOption(personalDetailList, report, draftComments, activeReportID)
         && (report.reportID !== activeReportID)
         && lodashGet(draftComments, `${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${report.reportID}`, '').length > 0;
 
+    const lastActorDetails = report ? personalDetailList.find(({login}) => login === report.lastActorEmail) : null;
```

Comment:
Please use the `_.find()` as we have standardized on using this for all array methods. Thanks!